browser/src/Utility.ts

```diff
@@ -245,3 +245,15 @@ export function ignoreWhilePendingPromise<T, U>(
 
     return ret
 }
+
+export function checkIfFileExistsSync(filename: string): boolean | Error {
```

Comment:
Would `fs.existsSync` work here instead?

Reply:
@bryphe I was initially going for something async and noted that `fs.exists` was being deprecated so assumed that was the same for `fs.existsSync`, but just checked and thats not the case so can tweak that, another perhaps over optimisation was that I wanted to ensure that the path not only existed but was  specifically a file on the *very* unlikely chance the function was used on a path that did exist but wasn't a file

Reply:
I see - if it has extra functionality, then I'm okay with keeping it in. Thanks for the explanation! That is confusing about  `fs.exists` being deprecated vs `fs.existsSync`.